content/api/commands/spy.md

```diff
@@ -38,11 +42,13 @@ The name of the `method` on the `object` to be wrapped.
 
 ### Yields [<Icon name="question-circle"/>](/guides/core-concepts/introduction-to-cypress#Subject-Management)
 
-Unlike most Cypress commands, `cy.spy()` is _synchronous_ and returns a value
-(the spy) instead of a Promise-like chain-able object.
-
-`cy.spy()` returns a [Sinon.js spy](https://sinonjs.org/releases/v6.1.5/spies/).
-All methods found on Sinon.JS spies are supported.
+- `cy.spy()` is a _utility function_, and is neither a command nor a query.
+  Cypress concepts like retries and timeouts do not apply.
+- `cy.spy()` is _synchronous_ and returns a value (the spy) instead of a
+  Promise-like chain-able object. It can be aliased.
+- `cy.spy()` returns a
+  [Sinon.js spy](https://sinonjs.org/releases/v6.1.5/spies/). All methods found
```

Comment:
I think this would be less useful  than the current link - that section basically just points users straight back to this page, without any details about the `spy` API.